### PR TITLE
Implement site page

### PR DESCRIPTION
**Pull request by salmaarja**

### CODE REVIEW

File: src/pages/Projects/index.jsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ const Projects = () => {
             setVisibleEdit(true)
             setEdit(e)
           }}
-          archiveRow={(e) => {
+          archiveProject={(e) => {
             setVisibleArchive(true)
```
Comment (MuhammadAw90) [inline src/pages/Projects/index.jsx:80]: it should stay archiveRow here

File: src/pages/Projects/index.jsx [hunk 2]
```diff
@@ -98,6 +98,9 @@ const Projects = () => {
               textDecoration: 'none'
             }}
             data={projects}
+            onClick={() => {
+              console.log('exporting')
+            }}
           >
```
Comment (MuhammadAw90) [inline src/pages/Projects/index.jsx:104]: You can remove
 
onClick={() => {
              console.log('exporting')
            }}

File: src/pages/Sites/index.jsx [hunk 3]
```diff
@@ -38,9 +70,47 @@ const Sites = () => {
...
-        <div className={style.export}></div>
+        <div className={style.export}>
+          <CSVLink
+            style={{
+              textDecoration: 'none'
+            }}
+            data={sites}
+            onClick={() => {
+              console.log('exporting')
+            }}
```
Comment (MuhammadAw90) [inline src/pages/Sites/index.jsx:81]: Also here, you can remove this,

 onClick={() => {
              console.log('exporting')
            }}

**Review by MuhammadAw90 — APPROVED** [review]
Great Job @salmaarja !

just checkout the comment "archiveRow" on Projects page...
you should keep it as archiveRow

**Review by tawfiknasser — APPROVED** [review]

**Pull request merged**